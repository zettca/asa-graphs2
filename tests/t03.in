8 11
8
3 6 -3
6 5 -2
3 5 1
2 8 -3
5 7 -2
2 4 -1
1 5 0
7 8 -1
4 5 3
1 6 -1
3 2 2
