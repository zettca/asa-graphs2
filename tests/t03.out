U
U
U
U
U
U
U
0
